eta-meeting-organizer-backend/src/main/java/hu/flowacademy/meetingorganizer/service/UserService.java

```diff
@@ -35,8 +35,9 @@ public void deleteUser(String id) {
     userRepository.deleteById(id);
   }
 
-  public User updateUser(String id, User user) {
-    user.setId(id);
-    return userRepository.save(user);
+  public User updateUser(String id) {
```

Comment:
Az nem gond, hogy így nem módosítható a felhasználó többi adata?

Reply:
Átmenetileg csak a validációhoz akarjuk használni. Tervben van egyedi username illetve company név, amit majd lehet változtatni, de az leghamarabb csak a jövőhéten lesz meg. Ahhoz lenne majd külön endpoint.